tower/src/builder/mod.rs

```diff
@@ -159,6 +159,15 @@ impl<L> ServiceBuilder<L> {
         self.layer(crate::util::option_layer(layer))
     }
 
+    /// Add a [`Layer`] built from a function that accepts a service and returns another service.
+    ///
+    /// See the documentation for [`layer_fn`] for more details.
+    ///
```

Comment:
nit/TIOLI: an example _might_ be nice, but the link to `layer_fn` is probably sufficient...